config/main-navigation.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * NOTE: code currently depends on Reference Docs being the first link
+ *
+ * :thinking: I wonder if a Context with knowledgeable of the current product being viewed
+ * (and updated by ProductSwitcher) would be able to programatically set internal paths
+ * in this array?
+ *   - example: Reference Docs `path` might be something like `/${currentProductSlug}/docs`
+ */
+const navigationLinks = [
+  {
+    id: 'reference-docs',
+    label: 'Reference Docs',
+    path: '/waypoint/docs',
+  },
+  {
+    href: 'https://www.waypointproject.io/',
+    id: 'overview',
+    label: 'Overview',
+  },
+  {
+    href: 'https://www.waypointproject.io/commands',
+    id: 'cli',
+    label: 'CLI',
+  },
+  {
+    href: 'https://www.waypointproject.io/plugins',
+    id: 'plugins',
+    label: 'Plugins',
+  },
+  {
+    href: 'https://www.waypointproject.io/downloads',
+    id: 'downloads',
+    label: 'Downloads',
+  },
+]
+
+export { navigationLinks }
```

Comment:
This will change per product I'm fairly sure, here's where we've been putting per-product data, as well as subnav items, which is what these would map to I think: https://github.com/hashicorp/dev-portal/blob/main/src/data/waypoint.json#L27

Reply:
Hmm true. I'll work on moving this

Reply:
@BRKalow Here's the change I made: https://github.com/hashicorp/dev-portal/pull/45/commits/415ace3beda06ff40130d4e21c41c06fd5028796. What do you think?